### PR TITLE
[DOCFIX] HBase docs structure improvement

### PR DESCRIPTION
**Pull request by LuQQiu**

### CODE REVIEW

File: docs/en/compute/Compute-HBase.md [hunk 1]
```diff
@@ -14,15 +14,18 @@ that you can easily store HBase tables into Alluxio at various storage levels.
...
-set up in accordance to these guides for either [Local Mode](Running-Alluxio-Locally.html) or
-[Cluster Mode](Running-Alluxio-on-a-Cluster.html).
-
-Please follow the guides for setting up HBase on
-[Apache HBase Configuration](https://hbase.apache.org/book.html#configuration).
-
-## Configuration
+* [Setup Java](Java-Setup.html) for Java 8 Update 60 or higher (8u60+), 64-bit.
+* An Alluxio cluster has been set up and is running according to either 
+[Local Mode](Running-Alluxio-Locally.html) or [Cluster Mode](Running-Alluxio-on-a-Cluster.html).
```
Comment (apc999) [inline docs/en/compute/Compute-HBase.md]: new way to specify the link 
`[link text]({{ site.baseurl }}{% link en/deploy/Running-Alluxio-Locally.md %})`

File: docs/en/compute/Compute-HBase.md [hunk 2]
```diff
@@ -14,15 +14,18 @@ that you can easily store HBase tables into Alluxio at various storage levels.
...
+* Make sure that the Alluxio client jar is available.
+This Alluxio client jar file can be found at `{{site.ALLUXIO_CLIENT_JAR_PATH}}` in the tarball 
+downloaded from Alluxio [download page](http://www.alluxio.org/download).
+Alternatively, advanced users can compile this client jar from the source code 
+by following the [instructions](Building-Alluxio-From-Source.html).
```
Comment (apc999) [inline docs/en/compute/Compute-HBase.md]: ditto

File: docs/en/compute/Compute-HBase.md [hunk 3]
```diff
@@ -69,39 +63,21 @@ Add the following properties to the same file `hbase-site.xml`.
 The last property is required to prevent HBase from flushing Alluxio file stream in a thread unsafe
```
Comment (apc999) [inline docs/en/compute/Compute-HBase.md]: I think the first two `fs.alluxio.impl` and `fs.AbstractFileSystem.alluxio.impl` is in fact not required. can you double check?

File: docs/en/compute/Compute-HBase.md (hunk 1 above)
Comment (LuQQiu) [reply docs/en/compute/Compute-HBase.md]: @apc999 have the `en/Java-Setup.md` already been removed?
Will change the links, thanks

Comment (apc999) [reply docs/en/compute/Compute-HBase.md]: no need to hyper link Jave setup any more. it is out of the scope of this documentation

**Comment by LuQQiu** [comment]
@apc999 Change to links to new format and remove the two fs.alluxio.impl and fs.AbstractFileSystem.alluxio.impl. They are not necessary in my local mode testing. 
PTAL, thanks!

File: docs/en/compute/Compute-HBase.md (hunk 3 above)
Comment (LuQQiu) [reply docs/en/compute/Compute-HBase.md]: Removed the two properties and restart HBase in local computer, examples finish successfully.

File: docs/en/compute/Compute-HBase.md (hunk 1 above)
Comment (LuQQiu) [reply docs/en/compute/Compute-HBase.md]: Removed Java setup, and changed the links

File: docs/en/compute/Compute-HBase.md [hunk 4]
```diff
@@ -14,15 +14,18 @@ that you can easily store HBase tables into Alluxio at various storage levels.
...
-The prerequisite for this part is that you have
-[Java](Java-Setup.html). Your Alluxio cluster should also be
-set up in accordance to these guides for either [Local Mode](Running-Alluxio-Locally.html) or
-[Cluster Mode](Running-Alluxio-on-a-Cluster.html).
-
-Please follow the guides for setting up HBase on
-[Apache HBase Configuration](https://hbase.apache.org/book.html#configuration).
-
-## Configuration
+* An Alluxio cluster has been set up and is running according to either
```
[the remark this answers is not in the document]
Comment (LuQQiu) [reply docs/en/compute/Compute-HBase.md]: I took a look at the alluxio user mailing list and found only a limited set of users reporting issues using Alluxio on HBase. There are not common issues. 

It seems that not that many users want to use Alluxio on HBase?

**Comment by aaudiber** [comment]
@apc999 There was still an outstanding issue

Currently the pre-requisites imply that HBase doesn't work with Alluxio when Alluxio is deployed on Mesos, Kubernetes, Yarn, or Docker. I don't think this limitation is correct, so we should remove it

**Comment by LuQQiu** [comment]
@aaudiber All the current compute framework documents contain that line:
```
An Alluxio cluster has been set up and is running according to either local mode or cluster mode.
```
Do you mean that this line implies that Compute frameworks don't work with Alluxio when Alluxio is deployed on Mesos, Kubernetes, Yarn, or Docker?

**Comment by aaudiber** [comment]
Right, the linked docs are two out of the many different deploy docs

**Pull request merged**